phpunit.xml.dist

```diff
@@ -11,7 +11,7 @@
     <ini name="error_reporting" value="-1"/>
     <env name="AWS_SHARED_CREDENTIALS_FILE" value="/dev/null" />
     <env name="AWS_CONFIG_FILE" value="/dev/null" />
-    <server name="SYMFONY_DEPRECATIONS_HELPER" value="max[indirect]=3" />
+    <server name="SYMFONY_DEPRECATIONS_HELPER" value="max[direct]=21&amp;max[indirect]=4" />
```

Comment:
? I'm not sure we should allow direct deprecation.

Reply:
They are deprecations for flysystem. "these method will be changed in version 2". We will never implement version 2.

Reply:
Is there another way to silent deprecation @nicolas-grekas ?

Like an a annotation saying? "This class will never be compatible with next major version, ignore this deprecation"